pkg/archive/archive_test.go

```diff
@@ -700,6 +701,56 @@ func tarUntar(t *testing.T, origin string, options *TarOptions) ([]Change, error
 	return ChangesDirs(origin, tmp)
 }
 
+func DecompressWithZstd(in []byte) []byte {
+	input := bytes.NewBuffer(in)
+	output := &bytes.Buffer{}
+	d, err := zstd.NewReader(input)
+	if err != nil {
+		return output.Bytes()
+	}
+	defer d.Close()
+	io.Copy(output, d)
+	return output.Bytes()
+}
+
+func CompressWithZstd(in []byte) []byte {
+	input := bytes.NewBuffer(in)
+	output := &bytes.Buffer{}
+	enc, _ := zstd.NewWriter(output)
+	io.Copy(enc, input)
+	enc.Close()
+	return output.Bytes()
+}
+
+func TestDetectCompression(t *testing.T) {
```

Comment:
Perhaps include `zstd` in the name here (e.g. `TestDetectCompressionZstd`), as that's currently the only compression we're detecting here.

Reply:
Done.